### PR TITLE
(nobug) - Update eslint-plugin-mozilla to 1.3.0 and ignore prettier for now

### PR DESCRIPTION
**Pull request by Mardak**
r? @dmose Ignore like most of m-c https://hg.mozilla.org/mozilla-central/rev/894a44c472727258338f2d013243c9b821a62686

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -38,7 +38,6 @@ module.exports = {
     // Remove this when m-c updates their eslint: See https://github.com/mozilla/activity-stream/pull/4219
     "RPMSendAsyncMessage": true,
     "NewTabPagePreloading": true,
-    "DOMLocalization": true,
```
Comment (dmose) [inline .eslintrc.js]: Why did this disappear?

**Comment by dmose** [comment]
Why did you add eslint-plugin-prettier, but not the other new peer dependencies added in the m-c commit you referenced?

File: .eslintrc.js (hunk 1 above)
Comment (Mardak) [reply .eslintrc.js]: It was added to eslint-plugin-mozilla as part of 1.3.0, so it's redundant here

**Comment by Mardak** [comment]
Why didn't I add `"eslint-config-prettier": "4.2.0",` ? The commit doesn't seem to require it (tests pass, no complaints from `npm i`) and I didn't add a script to use `eslint-config-prettier-check`

**Comment by Mardak** [comment]
`eslint-plugin-prettier` was added because otherwise eslint would complain that the plugin doesn't exist. And without`prettier`, the plugin would complain

**Pull request merged**